React/Views/RCTView.m

```diff
@@ -559,15 +559,25 @@ - (void)updateClippedSubviews
   }
 }
 
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+- (void)layout
+{
+  [super layout];
+  [self removeClippedSubviewsIfNecessary];
+}
+#else // TODO(macOS ISS#2323203)
 - (void)layoutSubviews
 {
+  [super layoutSubviews];
+  [self removeClippedSubviewsIfNecessary];
+}
+#endif // TODO(macOS ISS#2323203)
+
+- (void) removeClippedSubviewsIfNecessary {
   // TODO (#5906496): this a nasty performance drain, but necessary
   // to prevent gaps appearing when the loading spinner disappears.
   // We might be able to fix this another way by triggering a call
   // to updateClippedSubviews manually after loading
-
-  [super layoutSubviews];
-
   if (_removeClippedSubviews) {
```

Comment:
Should this be #if !TARGET_OS_OSX ?

Reply:
We call it above in layoutSubviews, which is iOS only